doghouse/service.go

```diff
@@ -0,0 +1,31 @@
+package doghouse
+
+// GitHub Check Request
+type CheckRequest struct {
+	// Installation ID of reviewdog GitHub APP.
+	// https://github.com/settings/installations/<INSTALLATION_ID>
+	// You can find from here: https://github.com/settings/installations/
+	InstallationID int `json:"installation_id,omitempty"`
+
+	SHA         string `json:"sha,omitempty"`
+	PullRequest int    `json:"pull_request,omitempty"`
+	Owner       string `json:"owner,omitempty"`
+	Repo        string `json:"repo,omitempty"`
+
+	// Name of the annotation tool.
+	// Optional.
+	Name        string        `json:"name,omitempty"`
+	Annotations []*Annotation `json:"annotations,omitempty"`
+}
+
+// GitHub Check Response
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported type CheckResponse should be of the form "CheckResponse ..." (with optional leading article)